lib/Model/Importer/Binary/PECOFFImporter.cpp

```diff
@@ -246,6 +314,9 @@ Error PECOFFImporter::import() {
   // Parse dynamic symbol table.
   parseImportedSymbols();
 
+  // Parse delay dynamic symbol table.
```

Comment:
What is a `delay dynamic symbol`? Is is a lazily resolved dynamic symbol?

Reply:
Yes, I've written this info in the commit message, but I will add a comment here as well.